libkbfs/md_ops.go

```diff
@@ -46,12 +46,19 @@ func (md *MDOpsStandard) convertVerifyingKeyError(ctx context.Context,
 }
 
 func (md *MDOpsStandard) verifyWriterKey(
-	ctx context.Context, rmds *RootMetadataSigned) error {
+	ctx context.Context, rmds *RootMetadataSigned, isTlfFinal bool) error {
 	if !rmds.MD.IsWriterMetadataCopiedSet() {
-		err := md.config.KBPKI().HasVerifyingKey(ctx,
-			rmds.MD.LastModifyingWriter,
-			rmds.MD.WriterMetadataSigInfo.VerifyingKey,
-			rmds.untrustedServerTimestamp)
+		var err error
+		if isTlfFinal {
+			err = md.config.KBPKI().HasUnverifiedVerifyingKey(ctx,
```

Comment:
Ah ok, we raced on the comment.  Put a comment somewhere about that?  The name is misleading.